src/registrar/templates/domain_dsdata.html

```diff
@@ -5,12 +5,16 @@
 
 
   {% block breadcrumb %}
-  {% if portfolio %}
   <!-- Navigation breadcrumbs -->
+   {% if portfolio %}
+    {% url 'domains' as url %}
+   {% else %}
+     {% url 'home' as url %}
+   {% endif %}
   <nav class="usa-breadcrumb padding-top-0" aria-label="Domain breadcrumb">
     <ol class="usa-breadcrumb__list">
       <li class="usa-breadcrumb__list-item">
-        <a href="{% url 'domains' %}" class="usa-breadcrumb__link"><span>Domains</span></a>
+        <a href="{{ url }}" class="usa-breadcrumb__link"><span>Domains</span></a>
```

Comment:
I know that I clicked on domains before and went to the main page but from this link, I just tried. It's actually not working. There's an issue when I click on 'Domains' from the non-org/non-enterprise model.
<img width="811" height="664" alt="Screenshot 2025-07-30 at 11 33 17 AM" src="https://github.com/user-attachments/assets/53947ac0-4628-47a1-86c5-2c920f1fef98" />